### DISCUSSION TITLE
Error while running OME on GPU

### DISCUSSION THREAD

**Discussion by jaggi-mohit** [OvenMediaLabs/OvenMediaEngine#1537] [Help]
Hi everyone,
I recently moved my OME from CPU to GPU as guided on https://airensoft.gitbook.io/ovenmediaengine/transcoding/gpu-usage but while publishing my UDP stream on OME it is raising errors.
FYI i am using FFmpeg to publish udp stream which further converted to webrtc (which was running fine in case of CPU) but when hardwareacceleration set to true causing trouble.

I am attaching snapshots FYR.

regards  

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/64644750/deaf10b0-5e1f-4ef0-9df6-7a046f8ef9ca)
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/64644750/da53afb6-7f3c-4606-9271-4a23b8a3e12c)

**Comment by Keukhan** [comment]
@jaggi-mohit 

Try running prerequisites.sh --enable-nvc again. The problem seems to be caused by the scale_npp filter not existing.

or, the problem may be that the OvenMediaEngine process references the libavfilter.so library from a different path. 
If you are referencing a library in a path other than opt/ovenmediaengine/lib as shown below, try removing the file.

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/21077363/8b07666a-e086-4378-b3f1-f6ec51adae46)

**Reply by jaggi-mohit** [reply]
Thanks for the reply!
I am sharing a snapshot of the  **ldd Ovenmediaengine**  command.
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/64644750/2c5f56bd-1e3b-47df-bdc1-06b7aca4d009)
Kindly let me know is there any issue in the dependencies as i believe mostly all dependencies are in " /opt " directory Moreover some dependecies are not present in my case.

**Reply by jaggi-mohit** [reply]
@Keukhan issue has been resolved once i have made changes in xml file. Thanks for the help
